Module export/import issues with carbon mapper

The same thing happened before with the DDF2 packages, so I guess the fix should be trivial. I'm getting:
```
ERROR in ./node_modules/@data-driven-forms/carbon-component-mapper/dist/esm/index.js
Module not found: Error: Can't resolve 'carbon-components-react/es/components/ButtonSet/ButtonSet.js' in '/home/skateman/Repositories/ManageIQ/manageiq-ui-classic/node_modules/@data-driven-forms/carbon-component-mapper/dist/esm'

```

--- a/packages/carbon-component-mapper/README.md
+++ b/packages/carbon-component-mapper/README.md
@@ -4,7 +4,7 @@
 
 [![Data Driven Form logo](https://raw.githubusercontent.com/data-driven-forms/react-forms/master/images/logo.png)](https://data-driven-forms.org/)
 
-carbon component mapper for [Data Driven Forms](https://github.com/data-driven-forms/react-forms).
+Carbon component mapper for [Data Driven Forms](https://github.com/data-driven-forms/react-forms).
 
 :book: For more information please visit the [documentation](https://data-driven-forms.org/). :book:
 

--- a/packages/carbon-component-mapper/rollup.config.js
+++ b/packages/carbon-component-mapper/rollup.config.js
@@ -36,7 +36,10 @@ const globals = {
   'prop-types': 'PropTypes',
   '@data-driven-forms/react-form-renderer': '@data-driven-forms/react-form-renderer',
   'carbon-components-react': 'carbon-components-react',
-  '@carbon/icons-react': '@carbon/icons-react'
+  '@carbon/icons-react': '@carbon/icons-react',
+  'carbon-components-react/lib/components/StructuredList/StructuredList': 'carbon-components-react/lib/components/StructuredList/StructuredList',
+  'carbon-components-react/lib/components/ProgressIndicator/ProgressIndicator':
+    'carbon-components-react/lib/components/ProgressIndicator/ProgressIndicator'
 };
 
 const babelOptions = {

--- a/packages/carbon-component-mapper/src/files/dual-list-select.js
+++ b/packages/carbon-component-mapper/src/files/dual-list-select.js
@@ -3,20 +3,15 @@ import PropTypes from 'prop-types';
 import DualListSelectCommon from '@data-driven-forms/common/src/dual-list-select';
 import clsx from 'clsx';
 
+import { Grid, Row, Column, Button, FormGroup, Search, TooltipIcon } from 'carbon-components-react';
+import { CheckmarkFilled16, ChevronRight32, ChevronLeft32, CaretSortDown32, CaretSortUp32 } from '@carbon/icons-react';
+
 import {
   StructuredListWrapper,
-  Grid,
-  Row,
-  Column,
-  StructuredListBody,
   StructuredListRow,
-  StructuredListCell,
-  Button,
-  FormGroup,
-  Search,
-  TooltipIcon
-} from 'carbon-components-react';
-import { CheckmarkFilled16, ChevronRight32, ChevronLeft32, CaretSortDown32, CaretSortUp32 } from '@carbon/icons-react';
+  StructuredListBody,
+  StructuredListCell
+} from 'carbon-components-react/lib/components/StructuredList/StructuredList';
 
 import './dual-list-select.scss';
 

--- a/packages/carbon-component-mapper/src/files/wizard.js
+++ b/packages/carbon-component-mapper/src/files/wizard.js
@@ -4,7 +4,9 @@ import clsx from 'clsx';
 
 import WizardCommon from '@data-driven-forms/common/src/wizard/wizard';
 import { FormSpy, WizardContext } from '@data-driven-forms/react-form-renderer';
-import { ProgressIndicator, ProgressStep, Button, Column, Grid, Row } from 'carbon-components-react';
+import { Button, Column, Grid, Row } from 'carbon-components-react';
+
+import { ProgressStep, ProgressIndicator } from 'carbon-components-react/lib/components/ProgressIndicator/ProgressIndicator';
 
 import './wizard.scss';
 

--- a/packages/common/babel.config.js
+++ b/packages/common/babel.config.js
@@ -201,14 +201,39 @@ const createCarbonCJSTransform = (env) => [
       },
       preventFullImport: false,
       skipDefaultConversion: false
-    }
-  },
-  `carbon-components-react-${env}`
-];
-
-const createCarbonIconsCJSTransform = (env) => [
-  'transform-imports',
-  {
+    },
+    'carbon-components-react/lib/components/StructuredList/StructuredList': {
+      transform: (importName) => {
+        let res;
+        const files = glob.sync(path.resolve(__dirname, `../../node_modules/carbon-components-react/${env === 'cjs' ? 'lib' : 'es' }/**/${carbonMapper(importName)}.js`));
+        if (files.length > 0) {
+          res = files[0];
+        } else {
+          throw new Error(`File with importName ${importName} does not exist`);
+        }
+        res = res.replace(path.resolve(__dirname, '../../node_modules/'), '');
+        res = res.replace(/^\//, '');
+        return res;
+      },
+      preventFullImport: false,
+      skipDefaultConversion: true
+    },
+    'carbon-components-react/lib/components/ProgressIndicator/ProgressIndicator': {
+      transform: (importName) => {
+        let res;
+        const files = glob.sync(path.resolve(__dirname, `../../node_modules/carbon-components-react/${env === 'cjs' ? 'lib' : 'es' }/**/${carbonMapper(importName)}.js`));
+        if (files.length > 0) {
+          res = files[0];
+        } else {
+          throw new Error(`File with importName ${importName} does not exist`);
+        }
+        res = res.replace(path.resolve(__dirname, '../../node_modules/'), '');
+        res = res.replace(/^\//, '');
+        return res;
+      },
+      preventFullImport: false,
+      skipDefaultConversion: true
+    },
     '@carbon/icons-react': {
       transform: (importName) => {
         let size = importName.match(/\d+/)[0];
@@ -229,7 +254,7 @@ const createCarbonIconsCJSTransform = (env) => [
       skipDefaultConversion: false
     }
   },
-  `@carbon/icons-react-${env}`
+  `carbon-components-react-${env}`
 ];
 
 module.exports = {
@@ -243,7 +268,6 @@ module.exports = {
         createBluePrintTransform('cjs'),
         createAntTransform('cjs'),
         createCarbonCJSTransform('cjs'),
-        createCarbonIconsCJSTransform('cjs')
       ]
     },
     esm: {
@@ -254,7 +278,6 @@ module.exports = {
         createBluePrintTransform('esm'),
         createAntTransform('esm'),
         createCarbonCJSTransform('esm'),
-        createCarbonIconsCJSTransform('esm')
       ]
     }
   }

